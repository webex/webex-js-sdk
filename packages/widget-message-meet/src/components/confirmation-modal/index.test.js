import React from 'react';
import renderer from 'react-test-renderer';

import ConfirmationModal from '.';

describe(`ConfirmationModal component`, () => {
  const messages = {
    title: `Delete`,
    body: `Are you sure you want to delete this message?`,
    actionButtonText: `Delete`,
    cancelButtonText: `Cancel`
  };

  const component = renderer.create(
    <ConfirmationModal messages={messages} />
  );

  it(`renders properly`, () => {
    expect(component).toMatchSnapshot();
  });
});
