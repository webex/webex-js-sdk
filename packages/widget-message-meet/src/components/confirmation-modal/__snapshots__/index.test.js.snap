exports[`ConfirmationModal component renders properly 1`] = `
<div>
  <div
    className="dialogue-modal">
    <div
      className="dialogue-modal-body">
      <div
        className="dialogue-modal-title-text">
        Delete
      </div>
      <div
        className="dialogue-modal-body-subtext">
        Are you sure you want to delete this message?
      </div>
      <button
        className="dialogue-modal-btn dialogue-modal-action-btn"
        onClick={undefined}
        title="Delete">
        Delete
      </button>
      <button
        className="dialogue-modal-btn dialogue-modal-exit-btn"
        onClick={undefined}
        title="Cancel">
        Cancel
      </button>
    </div>
  </div>
  <div
    className="dialogue-modal-backdrop" />
</div>
`;
